index.html

```diff
@@ -11,6 +11,9 @@
     <h1>
         Hello Microverse!
     </h1>
+    <p>
+        First of Everything
+    </p>
 
```

Comment:
Please ensure that you always add a short summary of changes introduced in the pull request. Please check details [HERE](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

![image](https://user-images.githubusercontent.com/10332499/159220481-f5b66d27-2eb3-408f-85f3-b8dba8efb909.png)

Reply:
- [ ] Checked done ✔️